### PR TITLE
Add Message Sender's Account Id to Mirror Node API Response

### PR DESCRIPTION
**Pull request by scalemaildev**
**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-0000-mirror-node-account-id.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+- hip: <HIP number (this is determined by the HIP editor)>
```
Comment (sergmetelin) [inline HIP/hip-0000-mirror-node-account-id.md:1]: Do you mind aligning it with the new HIP-1 and the new https://github.com/hashgraph/hedera-improvement-proposal/blob/master/hip-0000-template.md template?

The header should be reformatted to align with the table structure (remove `-` at each line in the header and wrap the header with `---` on top and bottom.

Please also make note that the review process has changed as well.

File: HIP/hip-0000-mirror-node-account-id.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- title: Message Account Id in Mirror Node API Response
+- author: scalemaildev
```
Comment (sergmetelin) [inline HIP/hip-0000-mirror-node-account-id.md:3]: Please adjust to the `author` field formatting referenced in HIP-1

File: HIP/hip-0000-mirror-node-account-id.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- type: Standards Track
+- category: Mirror
+- status: Draft
```
Comment (sergmetelin) [inline HIP/hip-0000-mirror-node-account-id.md:6]: Once the HIP is ready for review by the editors, please change the status to `Review`

File: HIP/hip-0000-mirror-node-account-id.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+- created: 2021-10-18
+- discussions-to: <a URL pointing to the official discussion thread>
```
Comment (sergmetelin) [inline HIP/hip-0000-mirror-node-account-id.md:8]: Please create a new discussion thread in the `Discussions` tab and link to it here

File: HIP/hip-0000-mirror-node-account-id.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+- updated: <comma separated list of dates>
```
Comment (sergmetelin) [inline HIP/hip-0000-mirror-node-account-id.md:9]: Please remove unused fields

File: HIP/hip-0000-mirror-node-account-id.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Many use cases for the HCS would want to know who is sending a message to a public topic. Workaround methods, such as querying the Transaction ID for a message and pulling the sender's Account ID from its timestamp, would involve extra API calls and put unnecessary strain on the mirror node and add complexity to the app. Having a trustworthy source of information for who sent a message would allow developers to handle logic involving the sender's ID in their application.
+
+## User stories
+
+As a developer, I want to know what account sent an HCS message from the message's metadata.
+  
+## Specification
+
+The technical specification should describe the syntax and semantics of any new features. The specification should be detailed enough to allow competing, interoperable implementations for at least the current Hedera ecosystem.
```
Comment (sergmetelin) [inline HIP/hip-0000-mirror-node-account-id.md:34]: Please fill out these sections to your best knowledge or leave them empty. Please note that the HIP can't be considered complete without Specification.

**Review by sergmetelin — CHANGES_REQUESTED** [review]

**Comment by scalemaildev** [comment]
I made the requested changes to the markdown file. Logs show this error:

```
2:28:58 PM: ────────────────────────────────────────────────────────────────
2:28:58 PM:   1. Build command from Netlify app                             
2:28:58 PM: ────────────────────────────────────────────────────────────────
2:28:58 PM: ​
2:28:58 PM: $ bundle exec jekyll build
2:28:58 PM: Could not locate Gemfile or .bundle/ directory
2:28:58 PM: ​
2:28:58 PM: ────────────────────────────────────────────────────────────────
2:28:58 PM:   "build.command" failed                                        
2:28:58 PM: ────────────────────────────────────────────────────────────────
2:28:58 PM: ​
2:28:58 PM:   Error message
2:28:58 PM:   Command failed with exit code 10: bundle exec jekyll build
2:28:58 PM: ​
2:28:58 PM:   Error location
2:28:58 PM:   In Build command from Netlify app:
2:28:58 PM:   bundle exec jekyll build
```

**Review by sergmetelin — APPROVED** [review]
LGTM

**Pull request merged**